src/test/java/com/microsoft/sqlserver/jdbc/parametermetadata/ParameterMetaDataWhiteSpaceTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Microsoft JDBC Driver for SQL Server
+ * 
+ * Copyright(c) Microsoft Corporation All rights reserved.
+ * 
+ * This program is made available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.jdbc.parametermetadata;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import com.microsoft.sqlserver.jdbc.SQLServerConnection;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+import com.microsoft.sqlserver.testframework.util.RandomUtil;
+
+@RunWith(JUnitPlatform.class)
+public class ParameterMetaDataWhiteSpaceTest extends AbstractTest {
+    private static final String tableName = "[" + RandomUtil.getIdentifier("ParameterMetaDataWhiteSpaceTest") + "]";
+
+    private static Statement stmt = null;
+
+    @BeforeAll
+    public static void BeforeTests() throws SQLException {
+        connection = (SQLServerConnection) DriverManager.getConnection(connectionString);
+        stmt = connection.createStatement();
+        createCharTable();
+    }
+
+    @AfterAll
+    public static void dropTables() throws SQLException {
+        stmt.execute("if object_id('" + tableName + "','U') is not null" + " drop table " + tableName);
```

Comment:
Please use `Utils.dropObjectIfExists(tableName,"IsTable",stmt)` function to drop table.